lib/install/config/webpack/development.server.js

```diff
@@ -10,6 +10,8 @@ module.exports = merge(devConfig, {
     host: devServer.host,
     port: devServer.port,
     compress: true,
+       headers: { "Access-Control-Allow-Origin": "*" },
+
```

Comment:
This helped me a lot! That line really should be there. I have problems loading Bootstrap fonts and it worked a few days ago on Rails 5.1.0.rc2, now on 5.1.0 it does not. Please fix and release a new version ASAP.